requirements.txt

```diff
@@ -16,3 +16,4 @@ Flask-SSLify==0.1.5
 alembic==0.8.3
 setuptools==36.0.1
 flask-secure-headers==0.6
+Flask-pyoidc==1.1.0
```

Comment:
Okay actually we aren't missing too many deps. I'll put in all the Flask-pyoidc deps for now and shore up the rest later.